Library/test-helpers/lib.sh

```diff
@@ -1376,6 +1376,10 @@ limeCreateTestPolicy() {
 
     # create policy.json...
     rlRun "limeCopyKeylimeFile --install cmd/convert_runtime_policy.py && python3 convert_runtime_policy.py -a allowlist.txt -e excludelist.txt -o policy.json"
+    # create signed policies and keys
+    rlRun "limeCopyKeylimeFile --install cmd/sign_runtime_policy.py && python3 sign_runtime_policy.py -r policy.json -p dsse-ecdsa-privkey.key -b ecdsa -o policy-dsse-ecdsa.json"
+    rlRun "limeCopyKeylimeFile --install cmd/sign_runtime_policy.py && python3 sign_runtime_policy.py -r policy.json -p dsse-x509-privkey.key -b x509 -o policy-dsse-x509.json"
```

Comment:
Would it be possible to deliver this script on a system using the common keylime installation script? This way we won't have to copy it here and moreover it would be included in code coverage measurement.

Reply:
@kkaarreell How would one do that? Do any scripts in [`setup.cfg`](https://github.com/keylime/keylime/blob/master/setup.cfg) work?

Reply:
I believe so. We are installing keylime through `python3 setup.py install`.

Reply:
I mean, I am assuming users would be using this script too and therefore it makes sense to install it but I may be wrong.

Reply:
@mbestavros I am sorry, I did more mess with my comments. I somehow confused `cmd` with `scripts`. We are installing `scripts` to `/usr/share/keylime/scripts` but this does not apply to `cmd` which we expect will be installed to `/usr/bin`. We can either leave it as it is now but if you expect that users would be using this script (I would) then I would propose to install it on a system through the setup script (and maybe consult other devs regarding other installation options). Also, I would consider renaming it (add it `keylime_` prefix and remove `.py` suffix) so the naming is consistent with other keylime scripts.

Reply:
@kkaarreell I've just tested manually building using `python setup.py install` and the signing script is accessible via `keylime_sign_runtime_policy`. So I can edit my tests to use that command.

Reply:
@kkaarreell I've updated `limeCreateTestPolicy` to use `keylime_sign_runtime_policy` in both new cases. I've also updated the conversion to use `keylime_convert_runtime_policy` using the same approach.